### ISSUE TITLE
Email for a user is not correctly formatted

### ISSUE THREAD

**Issue by stanislavefremov** [nilsteampassnet/TeamPass#4482]
### Steps to reproduce
1. Install the latest Teampass version
2. Create new user and wait for incoming email with user credentials for this user to be sent.

### Expected behaviour
Received email with credentials should be shown correctly formatted

### Actual behaviour
Email body is not shown as expected due to escaped HTML tags.

### Server configuration
**Operating system**:
Alpine Linux v3.18
**Web server:**
nginx/1.24.0
**Database:**
Mysql 5.7.44
**PHP version:**
8.2.7
**Teampass version:** 
3.1.2.157
**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**

### Client configuration
**Browser:**
No matter

**Operating system:**
No matter

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```
![image](https://github.com/user-attachments/assets/1c925270-187c-4ee1-8edd-5701cd17e044)

**Comment by stanislavefremov** [comment]
Suggested a fix in #https://github.com/nilsteampassnet/TeamPass/pull/4483

**Comment by corentin-soriano** [comment]
@nilsteampassnet can you close it as fixed by #4496?
